theseus/labs/lie/functional/so3_impl.py

```diff
@@ -258,36 +253,37 @@ def _jexp_impl(
         near_zero, torch.zeros_like(theta), (theta - sine) / theta3_nz
     )
 
+    batch = tangent_vector.shape[:-1]
```

Comment:
`batch` -> `batch_shape`? (but much better would be not to have duplicate code)

Reply:
Rename 'batch->size' as discussed offline.